scipy/stats/_entropy.py

```diff
@@ -27,10 +31,10 @@ def entropy(pk, qk=None, base=None, axis=0):
 
     Parameters
     ----------
-    pk : sequence
+    pk : array_like
         Defines the (discrete) distribution. ``pk[i]`` is the (possibly
         unnormalized) probability of event ``i``.
```

Comment:
I think this is good, except that I would want to add that the axis is decided according to axis argument. Wouldn't you agree?
Also I'm not sure if this will run for dimensions higher than 2, and even if it does  I dont think you can think f the result as entropy. Should the documentation reflect that?

Reply:
How do you find this?
```
For 1d array, defines the (discrete) distribution. ``pk[i]`` is the
(possibly unnormalized) probability of event ``i``. For 2d array, 
``pk`` defines a set of (discrete) distributions of some univariate 
 variables, along one of the axes. The axis is chosen according to 
 the ``axis`` argument.
```

Too verbose?

Reply:
How about nd arrays?

I don't think we need this level of detail in every function that supports an `axis` argument, as this behavior is familiar from many NumPy functions and probably every `scipy.stats` function that supports `axis` (e.g. `np.mean`, `scipy.stats.gmean`). 

Note also that gh-14651 intends to make this common across essentially all `scipy.stats` functions where [vectorization](https://en.wikipedia.org/wiki/Vectorization) makes sense. Perhaps as part of that, I will see if the stats tutorial needs an update to more carefully document behavior, but I don't think that needs to be in this PR.

Reply:
ok, so I'll the suggestion you provided, which is indeed much more concise. i'll add it to this PR.